__tests__/components/sections/homepage/OpenSource.test.tsx

```diff
@@ -0,0 +1,14 @@
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import { renderWithRedux } from "../../../../__mocks__/renderWithRedux";
+import OpenSource from "@/components/sections/homepage/OpenSource";
+
+describe("OpenSource", () => {
+  it("should render Open Source", () => {
+    renderWithRedux(<OpenSource />);
+    expect(screen.getByTestId("openSourceId")).toBeInTheDocument();
+    expect(screen.getByRole("link")).toBeInTheDocument();
+    expect(screen.getByRole("link").hasAttribute("href")).toBeTruthy();
+    expect(screen.getByRole("link").getAttribute("href")).toBe("https://github.com/dacadeorg/dacade-frontend-app");
```

Comment:
we can check if the link has the correct attribute `href` and right value in one go, look at the code i suggested above. hence, there's no need of this line